.github/dependabot.yaml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+- package-ecosystem: docker
+  directory: /
+  schedule:
+    interval: daily
```

Comment:
Do we want to update it daily? This will create PR and email spam for reviewers.

Please consider using the beta feature for grouping: https://github.com/kubernetes-csi/external-provisioner/blob/master/.github/dependabot.yaml#L2-L22

And also we should set a limit of the max PR it creates via [open-pull-requests-limit](https://github.com/kubernetes-csi/external-provisioner/blob/master/.github/dependabot.yaml#L36C3-L36C31)